ls1intum/ArTEMiS
# Bugfix/resume participation nullpointer

### Checklist

### Motivation and Context
Students see resume-buttons on exercises that cannot be paused and therefore not resumed.
This removes the unneeded resume-buttons and checks if exercise and participation exist when resuming.

### Description
When resuming an exercise, it now gets checked if the exercise and the participation exist, otherwise an error is shown to the user.

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Clean up the build plan of an user
4. The student should see "Resume exercise" now, keep that window open
5. Now as instructor, delete the whole participation.
6. As student, resume the exercise: An error message should be displayed.

### Screenshots
![image](https://user-images.githubusercontent.com/5084100/57876369-71c6a180-7815-11e9-9851-9e20170e6d83.png)
